### PR TITLE
Added a first draft of the sdTestEnvironmentShopware

### PR DESCRIPTION
**Pull request by sd-lueckel**

### CODE REVIEW

File: bin/runInSdTest.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+VERSION=$1
+
+if [ -z "${VERSION}" ]; then
+    echo "You must give a version to execute command on, for example 7.1 for PHP 7.1 container."
```
Comment (michz) [inline bin/runInSdTest.sh]: That not correct.

`echo "You must give a version to execute command on, for example 71 for PHP 7.1 container."`

(Notice the missing dot in version.)

File: etc/scripts/checkSdTestEnvironment.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+if [ ! -d ${PROJECT_DIR} ]; then
+    echo "Project directory not found in \$PROJECT_DIR. Did you call this script correctly? Cancel."
+    exit 1
+fi
+
+if [ ! -f ${PROJECT_DIR}/etc/test/docker-compose.yml ]; then
+    echo "No docker-compose.yml file found under ${PROJECT_DIR}/etc/test/docker-compose.yml - Needed for docker setup. Cancel."
+    echo "Perhaps forget to run 'vendor/bin/sdTest.sh init'?"
```
Comment (michz) [inline etc/scripts/checkSdTestEnvironment.sh]: forgot

File: README.md [hunk 3]
```diff
@@ -1,2 +1,106 @@
...
+
+    vendor/bin/sdTest.sh reset
+
+Executing a command in the testing environment
+----------------------------------------------
+
+Commands (for example to clear the cache or to run the setup) can be executed inside the container.
+You must give a version to execute command on, for example 7.1 for PHP 7.1 container:
+
+    vendor/bin/runInSdTest.sh 7.1 ./app/install.sh
```
Comment (michz) [inline README.md]: I think that won't work; See file `bin/runInSdTest.sh`

File: README.md [hunk 4]
```diff
@@ -1,2 +1,106 @@
...
+
+If you want to you can even get a shell inside the PHP container:
+
+    vendor/bin/runInSdTest.sh 7.1 /bin/bash
```
Comment (michz) [inline README.md]: I think that won't work; See file `bin/runInSdTest.sh`

**Review by michz — CHANGES_REQUESTED** [review]

**Comment by michz** [comment]
Generally: I think choosing nginx is a good idea as it is more lightweight than apache, but for real tests depending on shopware's frontend this will bring us trouble as no url rewriting is done up to now, or am I missing something?

**Comment by sd-lueckel** [comment]
nginx should be ok, we can add some configuration to ensure redirects etc. are working. I will look into it!

**Comment by sd-lueckel** [comment]
For a fully functional system this pull request (https://github.com/solutionDrive/docker-shopware-container/pull/4) is needed

**Review by michz — APPROVED** [review]

**Comment by michz** [comment]
solutionDrive/docker-shopware-container#4 is merged.

**Pull request merged**